evap/evaluation/models.py

```diff
@@ -1005,6 +1005,9 @@ def needs_login_key(self):
         return UserProfile.email_needs_login_key(self.email)
 
     def generate_login_key(self):
```

Comment:
maybe rename this to `ensure_login_key`? because now this does not always generate a new key.